playbooks/roles/node/tasks/install.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Install required system packages
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - build-essential
+    - libssl-dev
+
+- name: "Copy nvm file"
+  copy:
+    src: "install_nvm.sh"
+    dest: "install_nvm.sh"
+    mode: 0744
+  tags: node
+
+- name: "Install nvm"
+  become: false
+  shell: "bash install_nvm.sh"
+  tags: node
+  args:
+    executable: /bin/bash
+
+- name: Install node
+  become: false
+  shell: /bin/bash -c "source /home/{{ ansible_user }}/.nvm/nvm.sh && nvm ls-remote | grep 'Latest LTS'; nvm install {{ node_version }}"
```

Comment:
Is this necessary: 'nvm ls-remote | grep 'Latest LTS''?

Reply:
I followed the wiki for this. So need to check whether it is required in the wiki itself or not.
https://github.com/InterDigitalInc/AdvantEDGE/wiki/development-environment#step-4---install-nodejs-latest-lts-version

Reply:
In the manual procedure it is required to retrieve the latest version to install. Since we set the version to install we should not have to list the currently available versions.

Reply:
Ok removed